Footer 1480 kd

ktdee4 opened the pull request:
Initially created three footer regions, but after experimenting with placing blocks via the UI I found that enabling the existing Footer Menu and placing it in a footer region (via Structure--Block Layout) allowed me to add links (Structure--Menus--Footer--Add link) for the 'About', 'Get Involved', and 'Privacy' menu items based on the site design, that I could then style with one scss file. 
So I ultimately only created one footer region named footer-main, and then went ahead and created and saved a footer-main.scss file in the Components folder. I tested out enabling the Footer Menu with the three links again and it worked for me, so hopefully that can be recreated. 
If there was another reason for having three separate regions instead of just space for three links in a footer menu, I can easily add the two additional regions back in and adapt the styling.

JackG102 commented:
Hi Katie, Ashraf and I already answered some of your questions already in our Slack channel.  I also wanted to say here that your process of doing granular git commits and the comments are really clear.  I have no issues understanding your process based on the commits and the files edited in those commits.  Also, you PR message is clear and I understand what you did, any questions that we need addressed, etc.    Like we said in Slack, we do want three regions in the footer to place future blocks with information such as social media, main nav links, etc.

ktdee4 commented:
I added the other two regions back in so there should be three total now. I also updated the scss file to do display:flex and justify-content:space-around (applied to the .site-footer .container) to get the three regions to be on the same row and spaced apart. I also kept the styling I had done specifically for the Footer Menu, if that block is placed in the footer-right region. The font just needs to be made white.
